app/config/routing.yml

```diff
@@ -1,31 +1,39 @@
 wallabag_annotation:
   type : rest
   resource: "@WallabagAnnotationBundle/Resources/config/routing_annotations.yml"
+  prefix: "%base_path%/"
 
 wallabag_import:
     resource: "@WallabagImportBundle/Controller/"
     type: annotation
-    prefix: /import
+    prefix: "%base_path%/import"
 
 wallabag_api:
     resource: "@WallabagApiBundle/Resources/config/routing.yml"
-    prefix: /
+    prefix: "%base_path%/"
 
 app:
     resource: "@WallabagCoreBundle/Controller/"
     type: annotation
+    prefix: "%base_path%/"
 
 doc-api:
     resource: "@NelmioApiDocBundle/Resources/config/routing.yml"
-    prefix: /api/doc
+    prefix: "%base_path%/api/doc"
 
 rest :
     type : rest
     resource : "routing_rest.yml"
-    prefix : /api
+    prefix : "%base_path%/api"
+
+homepage_main:
+    path: "%base_path%/"
+    defaults:
+        _controller: WallabagCoreBundle:Entry:showUnread
+        page: 1
```

Comment:
I think so, yes. If /base_url/x/y matches a route, /base_url/ should match another route.  Personally, I wouldn't consider /base_url to be a valid point of access if it has further routes underneath and would rather not even forward /base_url to Wallabag. FOSUserBundle appears to work this way. However, the rest of Wallabag appears to be inconsistent, with parent routes /developer, /import/ and /import/pocket showing up in the routes list.

Whilst I could technically work around the problem with a URL rewrite in the web server, I would find it surprising that I would have to add such a rule, and it would probably be a hurdle for other users trying to set this up, which may cause more support queries, etc.